chia/server/server.py

```diff
@@ -493,29 +493,54 @@ async def start_client(
             connection_type_str = ""
             if connection.connection_type is not None:
                 connection_type_str = connection.connection_type.name.lower()
-            self.log.info(f"Connected with {connection_type_str} {target_node}")
-            if is_feeler:
+            if not is_feeler:
+                self.log.info(f"Connected with {connection_type_str} {target_node}")
+            else:
+                self.log.debug(f"Successful feeler connection with {connection_type_str} {target_node}")
                 asyncio.create_task(connection.close())
             return True
         except client_exceptions.ClientConnectorError as e:
-            self.log.info(f"{e}")
+            if not is_feeler:
+                self.log.info(f"{e}")
+            else:
+                self.log.debug(f"Feeler connection error. {e}")
```

Comment:
i'd probably reverse this to `if is_feeler`

Reply:
Thank you for the great suggestions!  I've made the modifications and resubmitted.